app/lint-baseline.xml

```diff
@@ -1,6 +1,19 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <issues format="5" by="lint 3.5.0" client="gradle" variant="all" version="3.5.0">
 
+    <issue
```

Comment:
This seems like a legit lint issue. Let's resolve. The point of the baseline is to not add new lint errors 😄

Reply:
Thanks for calling this out. It made me dig deeper on this one. See [this commit](https://github.com/twilio/twilio-video-app-android/pull/47/commits/2bc885953d9e18c7afeb217c44bcf779980b95ea) for the fix. I verified that overdraws is no longer occurring, but the lint error is still showing.

Reply:
I think you need to make the same change in this base theme `AppTheme.NoActionBar` then the lint error may be resolved.

Reply:
That didn't silence the lint error either. As you can see in the screenshot below, with "Debug GPU overdraw" enabled, the background is no longer highlighted after the fix applied to the AppTheme.
![overdraw](https://user-images.githubusercontent.com/2661383/75398527-274f7680-58b7-11ea-8ceb-bc9f29b608aa.png)

Reply:
Okay well thanks for trying!

Reply:
Of course. Thanks for pointing it out 👍